content/2016.09.jedit-trackers-with-kinto-en.md

```diff
@@ -0,0 +1,403 @@
+Title: Getting jEdit Trackers off the Ground using Kinto and Kinto.js
+Date: 2016-09-09
+Lang: en
+Url: jedit-trackers-with-kinto
+Slug: jedit-trackers-with-kinto
+Summary: My experience using Kinto and Kinto.js to browse jEdit tickets offline
+
+# Getting jEdit Trackers off the Ground using Kinto and Kinto.js
+
+This is a guest post by [Eric Le Lay](https://github.com/elelay) on his experience using Kinto.
+
+## Introduction
+
+Do you know [jEdit](http://jedit.org) — *programmer's text editor*?  
+It's a multi-platform text editor, written in java, with a myriad of plugins.
+I've been part of the development team for ten years.
+It's hosted on Sourceforge, with the code base under *subversion*.
```

Comment:
Subversion is written with a capital letter, unless you're talking about the command itself, which is `svn`. I probably wouldn't put it in italics, but I've seen this sort of style from other European writers so maybe it's OK.